api/v1/zz_generated.deepcopy.go

```diff
@@ -1,4 +1,3 @@
-//go:build !ignore_autogenerated
```

Comment:
This is required. Use Go 1.17 to generate changes.

Reply:
ack